fix: UIG-2897 - side-navigation - proza messages

ksvlaanderen opened the pull request:
data-vl-side-navigation-id attribuut toegevoegd als fix voor proza messages die niet renderen 
Storybook uitgebreid

[Jira](https://www.milieuinfo.be/jira/browse/UIG-2897)
[Bamboo](https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC308)

Goldflow commented on libs/elements/src/side-navigation/stories/vl-side-navigation.stories-doc.mdx:
```diff
@@ -53,6 +53,12 @@ Dit gebeurt op de volgende manier (we gebruiken content 1 als voorbeeld):
 2. Plaats op het VlSideNavigationToggle element het attribuut `data-vl-child="content-1"`
 3. Plaats op de child links het attribuut `data-vl-parent="content-1"`
 
+## Proza messages
```
Eventueel titel aanpassen naar "Gekende beperkingen" met subtitel `### Proza messages`
(zoals bv. [hier](https://github.com/milieuinfo/uigov-web-components/pull/280/commits/e2bbbb6ee26f3200ba5cfbe51fa382a85d9d26e2) )

ksvlaanderen replied:
Done

Goldflow commented on libs/elements/src/side-navigation/vl-side-navigation.element.ts:
```diff
@@ -31,10 +36,28 @@ export class VlSideNavigation extends BaseElementOfType(HTMLElement) {
+        */
+        if (sideNavigationId && VlSideNavigation.initializedSideNavigationId !== sideNavigationId) {
+            VlSideNavigation.initializedSideNavigationId = sideNavigationId;
             this._undress();
```
Mogelijks kan `this._undress();` hier weggelaten worden, gezien het de eerste keer is dat de side-nav zichzelf opbouwt, maar het zal niets breken om dit te laten staan.

ksvlaanderen replied:
Ik ga voor de zekerheid de this._undress() laten staan, ik heb het gevoel dat we anders problemen zouden kunnen hebben.

Goldflow replied:
Sounds good. I approve!

ksvlaanderen committed:
fix: UIG-2897 - side-navigation - proza messages

data-vl-side-navigation-id attribuut toegevoegd als fix voor proza messages die niet renderen
Storybook uitgebreid

kspeltix commented:
:tada: This PR is included in version 1.29.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.29.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
